test/tests/serial/user.bats

```diff
@@ -0,0 +1,42 @@
+load "${REPOSITORY_ROOT}/test/helper/common"
+load "${REPOSITORY_ROOT}/test/helper/setup"
+
+BATS_TEST_NAME_PREFIX='[User] '
+CONTAINER_NAME='dms-test_User_change_uid'
+
+function setup_file() {
+  _init_with_defaults
+
+  local CUSTOM_SETUP_ARGUMENTS=(
+    --env ENABLE_POP3=1
+    --env PERMIT_DOCKER=container
+    --env UID_DOCKER=10000
```

Comment:
Since this feature also provides `GID_DOCKER` that should be tested here too.

I've chosen `9042` instead just because it'd be a bit easier to grep for 🤷‍♂️ (_10000 is probably a bit too generic and could show up in logs or src elsewhere_)

```suggestion
    --env UID_DOCKER=9042
    --env GID_DOCKER=9042
```